react/src/base/links/SprkLink.stories.js

```diff
@@ -47,22 +45,40 @@ export const simple = () => (
 );
 
 export const iconWithTextLink = () => (
-  <SprkLink
-    element="a"
-    analyticsString="link-icon"
-    href="#nogo"
-    variant="has-icon"
-    idString="link-3"
-  >
-    <SprkIcon
-      additionalClasses="
+  <>
+    <SprkLink
+      element="a"
```

Comment:
I wonder, isnt the default element set to an `a`? I know this isnt related, but do we really need element="a" here on a SprkLink?